examples/DrawXNative/iOS/ViewController.cs

```diff
@@ -0,0 +1,116 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System;
+using SkiaSharp.Views;
+using UIKit;
+using Foundation;
+using DrawXShared;
+
+namespace DrawX.IOS
+{
+    public partial class ViewController : UIViewController
+    {
+        RealmDraw _drawer;
+
+        public ViewController(IntPtr handle) : base(handle)
```

Comment:
It will be, with some fixing up, when you build with a NuGet. Currently it is not used because the adjacent one in the _local_ build is used instead, but will fix up as much as possible.